_posts/2019-05-10-meet-us-at-barcelona.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Meet us at KubeCon 2019, Barcelona"
+description: Meet the community in person over the week 20-23th May at the KubeCon 2019 event in Barcelona. 
+date: 2019-04-18
+image: /images/meet-us-at-kubecon-2019-barcelona/backdrop.jpg
+categories:
+  - community
+  - workshop
+  - events
+  - kubernetes
+author_staff_member: alex
+dark_background: true
+---
+
+I'll outline a few places where you can find us if you'd like to catch a session, grab a coffee, swag or just chat about Kubernetes and Serverless
+
+### Monday 20th May - Serverless Practioners Summit
+
+The [Serverless Practioners Summit](https://spseu19.sched.com) is a co-located event hosted by the [CNCF](https://www.cncf.io) at KubeCon. It's on Monday so it's technically a "day 0" event. If you planned your travel accordingly, then we'd love to see you there. The event is being organised by the OpenFaaS Founder and the companies that are behind Knative such as IBM and Google.
```

Comment:
_It's happening on Monday (20th), so it's technically a "day 0" event_

The final sentence switches to third person, not sure if thats the intent since the blog starts with `I'll`

`behind Knative, such as IBM and Google.`